<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlicerParametersUI</class>
 <widget class="QDialog" name="SlicerParametersUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Slicer Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/ball.ico</normaloff>:/res/ball.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdApply">
       <property name="toolTip">
        <string>Apply Slicer to Selected Plots</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdClose">
       <property name="toolTip">
        <string>Close the dialog</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Slicer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Slicer Parameters </string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Slicer type:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="cbSlicer">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sector Interactor</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Annulus Interactor</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Box Interactor X</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Box Interactor Y</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wedge Interactor Q</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wedge Interactor Phi</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QTableView" name="lstParams">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Batch Slicing</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Batch Slicing Options </string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QListWidget" name="lstPlots">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbSave1DPlots">
            <property name="text">
             <string>Auto save generated 1D plots</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Files saved in:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtLocation">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cmdFiles">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>21</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>as </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbSaveExt">
                <item>
                 <property name="text">
                  <string>.txt</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>.xml</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>.h5</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="lblExtension">
                <property name="text">
                 <string>Append text:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtExtension">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Files will be saved as &amp;lt;SlicerType&amp;gt;&amp;lt;FileName&amp;gt;&amp;lt;AppendText&amp;gt;.txt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Fitting Options:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbFitOptions">
                <item>
                 <property name="text">
                  <string>No fitting</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fitting</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Batch fitting</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Inversion</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Batch inversion</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lstParams</tabstop>
  <tabstop>cbSlicer</tabstop>
  <tabstop>cmdApply</tabstop>
  <tabstop>cmdClose</tabstop>
  <tabstop>cmdHelp</tabstop>
  <tabstop>lstPlots</tabstop>
  <tabstop>cbSave1DPlots</tabstop>
  <tabstop>txtLocation</tabstop>
  <tabstop>cmdFiles</tabstop>
  <tabstop>cbSaveExt</tabstop>
  <tabstop>txtExtension</tabstop>
  <tabstop>cbFitOptions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
